contrib/endpoints/include/api_manager/api_manager.h

```diff
@@ -87,6 +87,12 @@ class ApiManager {
   virtual utils::Status GetStatistics(
       ApiManagerStatistics *statistics) const = 0;
 
+  // Returns the ApiManager initialization status.
+  //  - Code::UNKNOWN   Not initialized yet. The default value.
+  //  - Code::OK        Successfully initialized
+  //  - Code::ABORTED   Initialization was failed
+  virtual utils::Status InitializationStatus() = 0;
```

Comment:
The interface between Nginx and api_manager should be as simple as possible.  This should be done by request_handler::Check().

BTW,  if initial_status is for the status of api_manager constructor,   we don't need it.   We only care about the one called by Init().